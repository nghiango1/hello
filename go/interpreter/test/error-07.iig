// "type mismatch: INTEGER + BOOLEAN"
5 + true;
5;
