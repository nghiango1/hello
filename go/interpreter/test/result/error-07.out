ERROR: type mismatch: INTEGER + BOOLEAN
